### PR TITLE
Release notes for 3.2.0

### PR DESCRIPTION
**Pull request by robfalck**
### Summary

Updated release notes for 3.2.0

### Related Issues

- Resolves N/A

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
+**********************************
+# Release Notes for OpenMDAO 3.2.0
+
+July 20, 2020
+
+OpenMDAO 3.2.0 introduces significant changes in the way OpenMDAO works.
+The primary change is that the manual creation IndepVarComp outputs are no longer required because OpenMDAO will create them for you in the background.
+This was done in a backwards compatible way so that old models will still run and you can still use IndepVarComps if you want to.
```
Comment (Kenneth-T-Moore) [inline release_notes.md:8]: -> the manual creation of IndepVarComp outputs is no longer....

File: release_notes.md [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+- N2 adds a spinner to give the user an indication when it isn't finished rendering. #1485
+- All case recorder files now always contain system options information about all systems in the model. #1486
+- Model data now compressed when saved in N2 HTML file. #1490
+- Added support for om.slicer as a way to pass in slices to src_indices/indices arguments in add_design_var, add_response, add_constraint, promotes, and connect. #1491
+- Added flag for `under_approx` to let the user know when a system is operating under any derivative approximation scheme, and a new counter `iter_count_without_approx`. #1492
+- Automatically add a single, user-hidden, top-level IndepVarComp to provide outputs for every unconnected input in the model.
+
+## Bug Fixes:
+
+- Fix bug where exception was raised with while printing bounds violations via "print_bound_enforce" option on linesearch if one side of an output was unbounded. #1466
```
Comment (Kenneth-T-Moore) [inline release_notes.md:49]: with while -> while

File: release_notes.md [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+- Fixed an exception when using AddSubtractComp constructor instead of its add_equation method. #1474
+- Minor fixes for Appveyor CI. #1495
+- Problem.record() provides a useful error message if called before final_setup. #1545
+- Fixed a bug that was obliterating options passed to BalanceComp, and added tests for better coverage. #1546
+- Added quotes to install command in docs to prevent problems in some shell environments. #1547
+
 **********************************
```
Comment (Kenneth-T-Moore) [inline release_notes.md:56]: Missing: New Features
-------------------------
Allow the linear algebra components (DotProductComp, CrossProductComp, MatrixVectorProductComp, VectorMagnitudeComp) to each perform multiple calculations. #1503

File: release_notes.md (hunk 1 above)
Comment (Kenneth-T-Moore) [reply release_notes.md:8]: This comment still needs to be addressed.

Comment (swryan) [reply release_notes.md:8]: I'll make this change on the release branch.. coming in a minute..

**Comment by swryan** [comment]
Incorporated into #1552

**Pull request closed without merging**